.github/workflows/main.yml

```diff
@@ -60,12 +60,31 @@ jobs:
         name: owasp-report
         path: |
           target/dependency-check-report.html
+  native:
+    name: Java/Maven/Native
+    runs-on: ubuntu-latest
+    env:
+      MAVEN_PROJECTS: -pl :nessie-quarkus -pl :nessie-lambda
 
-    - name: Push Docker images
-      env:
-        DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
-      if: ${{ env.DOCKER_TOKEN }}
-      run: |
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup runner
+        uses: ./.github/actions/setup-runner
+      - name: Setup Java, Maven, Gradle
+        uses: ./.github/actions/dev-tool-java
+
+      - name: Build with Maven
```

Comment:
is this something that can be extracted as we're almost doing the same thing in the pull-request-native.yml?

Reply:
Rather not, because a composite-action is shown as a single step - i.e. the whole log for the build and the log for the relevant native step are unified. Having those as separate steps in the workflow-yaml is nicer.